detox/android/detox/src/main/java/com/wix/detox/DetoxActionHandlers.kt

```diff
@@ -110,3 +110,61 @@ class QueryStatusActionHandler(
             })
         }
 }
+
+class RecordingStateActionHandler(
+        private val instrumentsManager: DetoxInstrumentsManager,
+        private val wsClient: WebSocketClient
+) : DetoxActionHandler {
+
+    override fun handle(params: String, messageId: Long) {
+        val recordingPath = JSONObject(params).optString("recordingPath", null)
```

Comment:
While this works, the contract by `optString` says the fallback value shouldn't be a `null`. Could you consider using `has()` and `getString()` instead?

Reply:
has will check for key with any type of value instead of opting string type exactly.